packages/wallets/polkadotjs/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "description": "SwapKit Wallet - Polkadot.js",
+  "dependencies": {
+    "@polkadot/util": "12.6.2",
+    "@polkadot/util-crypto": "12.6.2",
+    "@swapkit/helpers": "workspace:*",
+    "@swapkit/toolbox-substrate": "workspace:*"
+  },
+  "devDependencies": {
+    "@polkadot/types": "12.1.1"
+  },
```

Comment:
Same as with talisman.
If needs to be included then update `build.ts` with:
```ts
import { buildPackage } from "../../../tools/builder";
import { dependencies, devDependencies } from "./package.json";

buildPackage({ dependencies: { ...dependencies, ...devDependencies } });
```

Reply:
We don't need to include the polkadot/types to each wallet package.

Reply:
done